pyhap/accessory.py

```diff
@@ -377,13 +381,42 @@ def setup_message(self):
         self.print_qr()
         print('Or enter this code in your HomeKit app on your iOS device: %s' % self.pincode.decode())
 
-    async def run(self, stop_event, loop=None):
+    def repeat(seconds):
```

Comment:
Maybe rename `every` or `interval`. `repeat` give the impression the decorated function will be called 3 times, then stop.

Reply:
Agree. `run_at_interval`?